lib/state/wallet.ts

```diff
@@ -321,33 +360,64 @@ if (initialWalletId) {
 export const useWallet = (): UseWallet => {
   const [userConfig, setUserConfig] = useAtom(userConfigAtom);
   const [walletState, setWalletState] = useAtom(walletAtom);
+  const [web3auth] = useAtom(web3authAtom);
+
+  const loadWeb3AuthWallet = async () => {
+    if (!web3auth) {
+      return;
+    }
+
+    web3auth.status === "not_ready" && (await web3auth.initModal());
+    await web3auth.connect();
+
+    if (web3auth.provider) {
+      const getKeypair = async (provider) => {
+        await cryptoWaitReady();
+        const privateKey = await provider.request({
+          method: "private_key",
+        });
+        const keyring = new Keyring({ ss58Format: 42, type: "sr25519" });
```

Comment:
Think the correct ss8 format is 73, same as you do in encoding of ztgAddress

Reply:
Ideally we should pull this from chain config, the property is `ss58Prefix`